site/docs/v1/tech/apis/_login-response-codes.adoc

```diff
@@ -7,17 +7,17 @@
 |The authentication was successful. The response will contain the User object that was authenticated.
 
 |202
-|The user was authenticated successfully. The user is not registered for the application specified by `applicationId` on the request.
+|The user was authenticated successfully. The user is not registered for the application specified by the `applicationId` on the request.
 The response will contain the User object that was authenticated.
 
 |203
 |The user was authenticated successfully. The user is required to change their password, the response will contain the `changePasswordId` to be used
 on the link:users#change-a-users-password[Change Password] API.  Since version `1.15.0`, the response will also contain a `changePasswordReason` field which can have one of the following values:
 
-* `Administrative`
-* `Breached`
-* `Expired`
-* `Validation`
+* `Administrative` - An administrator has required this user to change their password.
+* `Breached` - The password has been found to have belonged to a breached dataset and must be changed, per the Reactor configuration.
+* `Expired` - The password has expired, per the expiration setting in the tenant configuration.
+* `Validation` - The password fails the validation rules configured in the tenant password settings.
```

Comment:
I was thinking there was more than one spot these codes are documented. May want to do a quick search to be sure, otherwise looks great.

Reply:
You were right! I found another place and made notes as to why they couldn't be consolidated.